.github/release_notes_dependency_list.toml

```diff
@@ -12,7 +12,6 @@
 "androidx.lifecycle:lifecycle-runtime-ktx" = "[AndroidX Lifecycle Runtime](https://developer.android.com/jetpack/androidx/releases/lifecycle#{})"
 "androidx.lifecycle:lifecycle-viewmodel-compose" = "[AndroidX Lifecycle ViewModel Compose](https://developer.android.com/jetpack/androidx/releases/lifecycle#{})"
 "androidx.lifecycle:lifecycle-viewmodel-ktx" = "[AndroidX Lifecycle ViewModel](https://developer.android.com/jetpack/androidx/releases/lifecycle#{})"
-"androidx.preference:preference-ktx" = "[AndroidX Preference](https://developer.android.com/jetpack/androidx/releases/preference#{})"
```

Comment:
On line 39/40 this dependency is also mentioned. Should we delete it?

Reply:
My commit message is wrong, this line is duplicated in both lists but it's actually used by the example app so it should be kept in the excluded list